detekt/wrapper/src/main/kotlin/io/buildfoundation/bazel/rulesdetekt/wrapper/main.kt

```diff
@@ -45,8 +59,17 @@ fun main(rawArgs: Array<String>) {
                     detektStderr.flush()
                     detektStderr.close()
 
-                    originalStdout.println(detektStdoutFile.readText())
-                    originalStderr.println(detektStderrFile.readText())
+                    detektStdoutFile
+                            .bufferedReader(defaultCharset)
```

Comment:
You were right about encoding on Windows, fixed by explicitly specifying default one (which is what `PrintStream` does)

Also it's now streaming println instead of in-memory